### PR TITLE
BUGFIX: SD-508 handle overlap label for combo

### PR DESCRIPTION
**Pull request by lvkhanh**
Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [x] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [x] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [x] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [x] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Successful `extended test - examples`
- [x] Successful `extended test - storybook`
- [x] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/2341
- gdc-dashboard: https://github.com/gooddata/gdc-dashboards/pull/2116

# Related Jira tasks
- SD-508: https://jira.intgdc.com/browse/SD-508

### CODE REVIEW

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2281/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1651/ : UNSTABLE in 10m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5388/ : SUCCESS in 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4231/ : SUCCESS in 33m 25s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5387/ : SUCCESS in 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4234/ : FAILURE in 10m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5391/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4235/ : SUCCESS in 9m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5392/ : SUCCESS in 7s

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4238/ : SUCCESS in 11m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5395/ : SUCCESS in 15s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2284/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1653/ : SUCCESS in 12m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5396/ : SUCCESS in 2m 20s

**Comment by naibinh** [comment]
Please add storybook for it.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4252/ : SUCCESS in 8m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5412/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4254/ : SUCCESS in 7m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5415/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4260/ : SUCCESS in 7m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5424/ : SUCCESS in 11s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2294/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1662/ : SUCCESS in 13m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5425/ : SUCCESS in 9s

**Comment by naibinh** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5461/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 3s

**Comment by lvkhanh** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5463/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 2s

**Comment by lvkhanh** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4284/ : SUCCESS in 6m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5471/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4292/ : SUCCESS in 18m 01s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5484/ : SUCCESS in 9s

**Comment by lvkhanh** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2316/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1690/ : SUCCESS in 12m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5486/ : SUCCESS in 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4361/ : SUCCESS in 9m 29s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5586/ : SUCCESS in 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4369/ : SUCCESS in 4m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5604/ : SUCCESS in 7s

File: stories/core_components/ComboChart.tsx [hunk 1]
```diff
@@ -297,6 +304,29 @@ storiesOf("Core components/ComboChart", module)
             </ScreenshotReadyWrapper>,
         ),
     )
+    .add("stack primary measures has many data with 1 VIEW BY and chart type is COLUMN", () =>
+        screenshotWrap(
```
Comment (pbenes) [inline stories/core_components/ComboChart.tsx]: Just two small questions regarding the screenshot image for this storybook (https://github.com/gooddata/gooddata-react-components/pull/1067/files?short_path=9ac6f84#diff-9ac6f84d777981144e1f4ace43abb887) 
1. Is this image the case when without the fix there would be many overlapping labels? 
2. Is it correct that there are no labels now?

Comment (lvkhanh) [reply stories/core_components/ComboChart.tsx]: 1 & 2: Yes I add story book image base on use case in bug, for column-column if label is overlap We will hidden

Comment (pbenes) [reply stories/core_components/ComboChart.tsx]: Thanks :+1:

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2358/ : SUCCESS in 11m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4380/ : SUCCESS in 17m 37s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1111/ : SUCCESS in 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1378/ : SUCCESS in 34m 01s

**Pull request merged**